src/components/Popover/Popover.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import { ReactComponent as Close } from '../../assets/icons/Close.svg';
+import { IconButton } from '../IconButton';
+import { Text } from '../Text';
+import { TPAComponentsConsumer } from '../TPAComponentsConfig';
+import { POPOVER_DATA_HOOKS, POPOVER_DATA_KEYS } from './dataHooks';
+import styles from './Popover.st.css';
+
+export interface PopoverProps {
+  title?: string;
+  onClose(): void;
+  withArrow?: boolean;
+  rightArrow?: boolean;
+  arrowTop?: string;
```

Comment:
I think we should also accept `number` for the `arrowTop`

Reply:
In addition, i'm still unsure about this API.
I feel that have one prop, such as `arrowPosition` would be a bit more intuitive.
Something like:
```js
type ArrowPosition = {
  side: 'left' | 'right';
  top: string | number;
}
...
arrowPosition: ArrowPosition;
```

or something like that

Reply:
i think about arrowTop: number and side: sidesEnum